pkgs/games/quake3/quake3e/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, curl, libGL, libX11, libXxf86dga, alsaLib, libXrandr, libXxf86vm, libXext, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "Quake3e";
+  version = "2019-09-09";
+
+  src = fetchFromGitHub {
+    owner = "ec-";
+    repo = pname;
+    rev = version;
+    sha256 = "0i9flw8h87lagdpbci6vgqkriv05p3bidgqb4pwrxls947zwfcw8";
+  };
+
+  buildInputs = [ curl libGL libX11 libXxf86dga alsaLib libXrandr libXxf86vm libXext ];
+  enableParallelBuilding = true;
+
+  patchPhase = ''
```

you may want to move this to postPatch, as the default patchPhase is helpful with scripts and a few other things (this probably doesn't matter as it seems to build fine).
```suggestion
  postPatch = ''
```